searchcore/src/tests/proton/attribute/attribute_manager/attribute_manager_test.cpp

```diff
@@ -395,7 +395,7 @@ TEST_F("require that predicate attributes are flushed and loaded", BaseFixture)
         uint32_t doc_id;
         a1->addDoc(doc_id);
         index.indexEmptyDocument(doc_id);
-        pa.commit(100, 100);
+        pa.commit(CommitParam(100, true));
```

Comment:
Why are we using _forceUpdateStats=true_ here? That is not equal to the previous behaviour. The same comment applies for other changes in this file, attribute_test.cpp and attributeflush_test.cpp.

Reply:
Removed semantic change